aws/requests.yaml

```
@@ -9,6 +9,10 @@ releases:
       requests:
         - name: containerlinux
           version: "<= 2905.2.6"
+    - name: "> 16.3.0 > 16.1.0 > 16.0.1"
```

Can we change it to 16.3.1? We have a small fix for EC2 instances which shouldn't have any other changes? Pretty please?
```suggestion
    - name: "> 16.3.1 > 16.1.0 > 16.0.1"
```

---

sure :)